src/Rx.ts

```diff
@@ -110,7 +110,7 @@ import {ConnectableObservable} from './observable/ConnectableObservable';
 import {Notification} from './Notification';
 import {EmptyError} from './util/EmptyError';
 import {ArgumentOutOfRangeError} from './util/ArgumentOutOfRangeError';
-import {nextTick} from './scheduler/nextTick';
+import {ObjectUnsubscribedError} from './util/ObjectUnsubscribedError';
```

Comment:
Build failed by removing importing `nextTick` in here still being used in line 121.